module/x/gravity/types/test_utils.go

```diff
@@ -0,0 +1,50 @@
+package types
+
+import (
+	"encoding/hex"
+	"math/big"
+	"math/rand"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
```

Comment:
These functions perform type massaging where necessary between the `math/rand` library and the sdk types / go native types.